robottelo/host_helpers/cli_factory.py

```diff
@@ -1151,35 +1151,6 @@ def setup_cdn_and_custom_repos_content(
                     'content-view-id': content_view['id'],
                 }
             )
-        # Get organization subscriptions
-        subscriptions = self._satellite.cli.Subscription.list(
```

Comment:
Since we are removing this, just make sure that all the tests in robottelo using this function is converted to use SCA only manifest else they will fail.

Reply:
noted. I will address this in another PR when I clean up fixutres and entitlement functionality